File: Easy/reverse.py
# https://edabit.com/challenge/9hQogtkbZSSJ8tYsG
def reverse(txt):
	return txt.swapcase()[::-1]

print(reverse("Hello World"), "DLROw OLLEh")
print(reverse("ReVeRsE"), "eSrEvEr")
print(reverse(""), "")
print(reverse("Radar"), "RADAr")

File: javascript/Very_Easy/reverse.js
// https://edabit.com/challenge/noqGmqk6StwApSGwa

function reverse(bool) {
	return typeof bool == 'boolean' ? !bool : "boolean expected";
}

// https://edabit.com/challenge/TiAz6mfHTtkoqMszA

function reverse(str) {
	return str==false?'':reverse(str.substr(1))+str[0];
}

console.log(reverse("hello"), "olleh")
console.log(reverse("world"), "dlrow")
console.log(reverse("a"), "a")
console.log(reverse(""), "", "An empty string should return an empty string.")